src/pages-content/pricing-data.js

```diff
@@ -56,7 +56,7 @@ const faqs = [
     {
         q: 'Can I switch between PostHog Cloud and Self-hosted plans?',
         a:
-            'You can switch from PostHog Cloud to Self-hosted Scale - just email our support team (hey@posthog.com). We will manually transfer your data, free. Likewise, you can switch from Self-hosted Scale to PostHog Cloud too - email us.',
+            'You can switch from PostHog Cloud to Self-hosted Scale and vice versa - just email our support team (hey@posthog.com). We will manually transfer your data, free. If you are switching between PostHog Cloud and Self-hosted Free then you can make the transfer yourself using [our plugin](https://github.com/PostHog/posthog-plugin-migrator3000).',
```

Comment:
> Self-hosted Free

Thoughts on saying _Self-hosted Open source_ instead? We don't list "Free" as an edition name on our site, so it feels like this should match a plan "name".

Reply:
Agree - I was just following the question below in the FAQ which says 'Self-hosted Free'. Will update!